addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -1,123 +1,34 @@
-.o_pivot {
-    .o_pivot_cell_value {
-        font-size: 1em;
-        .o_comparison {
-            font-size: 0.61em;
-        }
-        .o_variation {
-            &.o_positive {
-                color: green;
-            }
-            &.o_negative {
-                color: red;
-            }
-        }
+.o_pivot table {
+    // Temporary solution (waiting the conversion of Dashboard view to OWL) to keep the width
```

Comment:
dashboard view is converted to owl

Reply:
Hello @aab-odoo 👋 ,
Thanks to @stefanorigano to helped me to find a way to remove this line. Now there's a new prop to detect if the view is embedded. Depending of the context, the Pivot table has directly the `w-auto` or `w-100` class.

Related Enterprise PR : [https://github.com/odoo/enterprise/pull/23701](https://github.com/odoo/enterprise/pull/23701)